packages/account/src/Components/poi/status/unsupported/card-details/utils.ts

```diff
@@ -1,21 +1,21 @@
 import { localize } from '@deriv/translations';
 
-export const setInitialValues = fields => {
+export const setInitialValues = (fields: []) => {
     const values = {};
-    fields.forEach(field => {
+    fields.forEach((field: { name: string }) => {
```

Comment:
can we define this type outside and use it also on fields definition on line 3?

Reply:
sure will edit it 🙏